jupyter-book/deconvolution/deconv_covid_MuSiC_v0.ipynb

```diff
@@ -0,0 +1,2416 @@
+{
```

Comment:
<p>Do we need to match the Seurat visualization? I guess we can just get rid of this step, also du you want to use the highly variable genes for calculating the PCA space? I would maybe also remove the PCA plot, if we are not showing something on it. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/93/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>So maybe we can remove the whole visualization part. It's not really part of the deconvolution and maybe it's confusing...</p>

Reply:
<p>yes we are using HV (by default, no?). Yes we can remove PCA visualisation</p>